src/pull-request-core.ts

```diff
@@ -60,18 +64,36 @@ class Client {
             destinationRepo = sourceRepo
         }
 
-        const { data: pullResponse } = await this.octokit.pulls.create({
-            owner: destinationOwner,
-            repo: destinationRepo,
-            title: subject,
-            body: description,
-            head: sourceBranch,
-            base: destinationBranch,
-            maintainer_can_modify: maintainerCanModify,
-            draft,
-        })
+        try {
+            const { data: pullResponse } = await this.octokit.pulls.create({
+                owner: destinationOwner,
+                repo: destinationRepo,
+                title: subject,
+                body: description,
+                head: sourceBranch,
+                base: destinationBranch,
+                maintainer_can_modify: maintainerCanModify,
+                draft,
+            })
 
-        return pullResponse
+            return pullResponse
+        } catch (e) {
+            if (e.status && e.status === 422) {
```

Comment:
The `e.status` here is not useful because if the property is not set, it would just be `undefined`, which is `!== 422`

Reply:
Ah, right